src/components/shared/inputs/Pay/MemoFormItem.tsx

```diff
@@ -0,0 +1,61 @@
+import { t } from '@lingui/macro'
+import { Form, Input } from 'antd'
+import { SmileOutlined } from '@ant-design/icons'
+import BannyAttachModal from 'components/shared/modals/BannyAttachModal'
+import { useState } from 'react'
+
+export default function MemoFormItem({
+  value,
+  onChange,
+}: {
+  value: string
+  onChange: (memo: string) => void
+}) {
+  const [bannyAttachModalVisible, setBannyAttachModalVisible] =
+    useState<boolean>(false)
+  return (
+    <>
+      <Form.Item
+        label={t`Memo`}
+        name="memo"
+        className={'antd-no-number-handler'}
+      >
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            position: 'relative',
+          }}
+        >
+          <Input.TextArea
+            placeholder={t`(Optional) Add a memo to this payment on-chain`}
```

Comment:
**suggestion** not related to your PR, but generally don't wan't important explainer info like this in placeholders. Can we try this in `extra` prop?

Reply:
Doneski